### ISSUE TITLE
What environment variable should be used to add a java.library.path

### ISSUE THREAD

**Issue by warmachine2017** [fabric8io-images/run-java-sh#89]
Good day!

I am trying to deploy and run a spring boot application in a kubernetes cluster using fabric8 maven plugin. Now, I have a third party library path which I need to specify when running my application. I've tried JAVA_OPTIONS and JAVA_LIB_DIR but they did not work. Any idea where should I put the said configuration? "-Djava.library.path=/path/to/libs"

Thanks!

**Comment by sparty02** [comment]
I came across this need recently too.  @rhuss  would you be open to a PR to add support for something like:

```bash
java_native_library_options() {
  if [ -n "${JAVA_NATIVE_LIBRARY_PATH:-}" ]; then
    echo "-Djava.library.path=$JAVA_NATIVE_LIBRARY_PATH"
  fi
}
```

**Comment by rhuss** [comment]
yes, sure ! Happy to help to integrate such a PR, but not sure how much time budget i can spend (so might be a bit slow).
